### ISSUE TITLE
[Utility] Account Logic - Send and Receive Funds - Add Tx Fees to Mempool

### ISSUE THREAD

**Issue by jessicadaugherty** [pokt-network/pocket#326]
## Objective

Add tx fees from all send txs to mempool. Blocked by https://github.com/pokt-network/pocket/issues/325

## Origin Document

The foundations for sending transactions were built out [here](https://github.com/pokt-network/pocket/blob/main/utility/transaction.go) but they do not include adding tx fees to the mempool.

## Goals

- [ ] Add more production ready functionality for sending E2E txs in localnet and devnet

## Deliverable

- [ ] Move tx fees with funds to mempool
- [ ] Store tx fees with funds in mempool

## Non-goals / Non-deliverables

- Implementing "unhappy path" for save points

## General issue deliverables

- [ ] Update the appropriate CHANGELOG
- [ ] Update any relevant READMEs (local and/or global)
- [ ] Update any relevant global documentation & references
- [ ] If applicable, update the source code tree explanation
- [ ] If applicable, add or update a state, sequence or flowchart diagram using [mermaid](https://mermaid-js.github.io/mermaid/)

## [Optional] Testing Methodology

- **Task specific tests**: `make ...`
- **All tests**: `make test_all`
- **LocalNet**: verify a `LocalNet` is still functioning correctly by following the instructions at [docs/development/README.md](https://github.com/pokt-network/pocket/tree/main/docs/development)

---

**Creator**: @jessicadaugherty

**Comment by Olshansk** [comment]
@jessicadaugherty Given the updated description of #325, I believe we can close this out as a duplicate. Wdyt?

**Comment by jessicadaugherty** [comment]
@Olshansk aligned
